examples/circuitbreaker.xml

```diff
@@ -0,0 +1,41 @@
+<policies>

```

Comment:
I think we can remove this, circuit breaker is built-in nowadays - https://learn.microsoft.com/en-us/azure/api-management/backends?tabs=bicep#circuit-breaker-preview

Reply:
removed the file, and updated the readme as well.